helm/cluster-openstack/templates/_kubeadm_control_plane_template.tpl

```diff
@@ -14,6 +14,12 @@ spec:
           apiServer:
             extraArgs:
               cloud-provider: external
+              {{- if .Values.oidc.enabled }}
+              oidc-issuer-url: https://dex.{{ include "resource.default.name" $ }}.{{ .Values.baseDomain }}
```

Comment:
Getting this from the `giantswarm`/`giantswarm-test` appcatalog defaults. See https://github.com/giantswarm/installations/pull/1952 for an example.